core/domain/stats_jobs_continuous.py

```diff
@@ -157,20 +147,6 @@ def reduce(key, stringified_values):
             for v, interaction_ids in versioned_interaction_ids.iteritems()
         }
 
-        # Verify all interaction ID and item ID containers are well-structured.
-        for version, interaction_ids in versioned_interaction_ids.iteritems():
```

Comment:
Only one interaction id is returned in map

Reply:
I am not sure what you mean here. This is a reduce() method. It aggregates the results from different map() calls.

Reply:
Doesn't reduce work on a single exploration at a time?

Reply:
Yes, but an exploration has multiple interaction IDs (one per state), right?

Reply:
I am sorry but the job maps over `StateAnswersModel`, isn't it? And one model instance will contain only one interaction_id, won't it?

We are iterating over `versioned_interaction_ids` and `versioned_interaction_ids` is in the form of {version: interaction_ids} but interaction_ids would only contain one interaction_id since we are yielding only one interaction_id for one exp version

Reply:
Sorry I was able to test it. Reverted this!